lib/react_client.dart

```diff
@@ -626,116 +893,8 @@ ReactDartComponentFactoryProxy _registerComponent(
   return new ReactDartComponentFactoryProxy(reactComponentClass);
 }
 
-class ReactJsContextComponentFactoryProxy extends ReactJsComponentFactoryProxy {
```

Comment:
This wasn't removed, it was just moved up to group it with other [FactoryProxy]s